python_src/src/lamp_py/performance_manager/l0_gtfs_rt_events.py

```diff
@@ -143,6 +143,10 @@ def upload_to_database(
     already in the database and those that are brand new. update preexisting
     events where appropriate and insert the new ones.
     """
+    # handle empty events dataframe
+    if events.shape[0] == 0:
+        return
+
```

Comment:
handle empty dataframe without error

Reply:
why does this need to be handled? this is the statement thats putting this function over the pylint limit (and causing the test to fail)

Reply:
We do still have a lot of erroneous logging statements that we could probably pull out at this point, to get us under the limit for pylint.